app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="hyperionics.com.standouttest">
```

Comment:
restore package name back to wei.mark.example. Then you can refer to the service below with `.MostBasicWindow` like before